cloudlift/deployment/service_template_generator.py

```diff
@@ -16,6 +16,7 @@
                              PortMapping, Service, TaskDefinition)
 from troposphere.elasticloadbalancingv2 import Action, Certificate, Listener
 from troposphere.elasticloadbalancingv2 import LoadBalancer as ALBLoadBalancer
+from troposphere.elasticloadbalancingv2 import LoadBalancer as NLBLoadBalancer
```

Comment:
we are doing the same import as a different name, aws in cloud formation doesn't differentiate as a different resource, nlb is one time of elbv2, where alb is another type. probably it's better to refactor code to use it as LoadBalancer instead of specific type.

Reply:
Good point my friend.  Will update.

Reply:
Changed to `from troposphere.elasticloadbalancingv2 import LoadBalancer as LoadBalancerV2`, still avoids collision with `troposphere.ecs.LoadBalancer`.